src/components/Hero.jsx

```diff
@@ -24,10 +25,11 @@ const Hero = () => {
               </p>
             </p>
 
-            <button class="btn btn-warning-ghost ff-btn-primary btn-lg">
-              START NOW
-            </button>
-
+            <Link to="/videolistingpage">
+              <button class="btn btn-warning-ghost ff-btn-primary btn-lg">
+                START NOW
+              </button>
+            </Link>
```

Comment:
Can shift to Navlinks later on as they have an isActive attribute enabling you to style it whenever the user visits the component